terraform/task-definitions/publisher.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "name": "publisher",
+    "image": "govuk/publisher:serve-assets-in-prod",
+    "essential": true,
+    "environment": [
+      { "name": "ASSET_HOST", "value": "www.gov.uk" },
+      { "name": "BASIC_AUTH_USERNAME", "value": "gds" },
+      { "name": "EMAIL_GROUP_BUSINESS", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_CITIZEN", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_DEV", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_FORCE_PUBLISH_ALERTS", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "FACT_CHECK_SUBJECT_PREFIX", "value": "dev"},
+      { "name": "FACT_CHECK_USERNAME", "value": "govuk-fact-check-test@digital.cabinet-office.gov.uk"},
+      { "name": "GOVUK_APP_DOMAIN", "value": "test.govuk-internal.digital" },
+      { "name": "GOVUK_APP_NAME", "value": "publisher"},
+      { "name": "GOVUK_APP_TYPE", "value": "rack"},
+      { "name": "GOVUK_STATSD_PREFIX", "value": "fargate" },
+      { "name": "GOVUK_ASSET_ROOT", "value": "https://assets.digital.cabinet-office.gov.uk" },
```

Comment:
Probably better to use a non-prod address here (maybe assets.test.publishing.service.gov.uk?), even though it's not going to modify any assets either way.